math/FunMath.cpp

```diff
@@ -1 +1,14 @@
 #include "FunMath.h"
+
+
+int NextLeapYear(int current_year) {
+    int curr = current_year;
+    while (true) {
+        if ((curr % 4 == 0 && curr % 100 != 0) || (curr % 4 == 0 && curr % 100 == 0 && curr % 400 == 0)) {
```

Comment:
`curr % 100` is somewhat equivalent as `curr % 400`, so just keep `curr % 400`

Reply:
Okay, I'll go fix it